pandas_gbq/gbq.py

```diff
@@ -1122,12 +1128,14 @@ def to_gbq(
             )
         elif if_exists == "replace":
             connector.delete_and_recreate_table(dataset_id, table_id, table_schema)
-        elif if_exists == "append":
+        else:
```

Comment:
There's already a check above for known `if_exists`, so this was an impossible branch.